types/vis/index.d.ts

```diff
@@ -107,8 +107,11 @@ export interface DataGroup {
   style?: string;
   subgroupOrder?: string | (() => void);
   title?: string;
-  nestedGroups?: number[];
+
```

Comment:
```suggestion
```

Reply:
@MichaelBitard: Should this be fixed before I merge?

Reply:
It was a suggestion and I already approved the PR, so it was up to you ;)